integration/e2e/utils.go

```diff
@@ -0,0 +1,85 @@
+package e2e
+
+import (
+	"github.com/hyperledger/fabric/integration/nwo"
+)
+
+type ChaincodeQueryWithHex struct {
+	ChannelID string
+	Name      string
+	Ctor      string
+}
```

Comment:
You don't need this struct and its corresponding methods. We needed it so we could get hex encoded output. So you can remove that piece.

Reply:
fixed